app/tests/unit/components/Form/ProjectContactForm.test.tsx

```diff
@@ -323,12 +336,14 @@ describe("The ProjectContactForm", () => {
     componentTestingHelper.loadQuery();
     componentTestingHelper.renderComponent();
     screen.getByText(/submit/i).click();
-    expect(
-      componentTestingHelper.environment.mock.getMostRecentOperation().request
-        .variables.input
-    ).toMatchObject({
-      formChangePatch: { changeStatus: "staged" },
-    });
+    const allOperations =
+      componentTestingHelper.environment.mock.getAllOperations();
+    expect(allOperations.length).toEqual(5); // first operation is special; next four are form_change
```

Comment:
I know "special" isn't the right word but I can't remember what @pbastia called it

Reply:
The first operation returned by `allOperations` is the query itself